mbed_tools_ci_scripts/utils/aws_helpers.py

```diff
@@ -0,0 +1,117 @@
+"""Helpers for interacting with AWS services such as S3.
+
+Based on the Python SDK called BOTO https://aws.amazon.com/sdk-for-python/.
+"""
+import boto3
+import logging
+import os
+from pathlib import Path
+import mimetypes
+from mbed_tools_ci_scripts.utils.configuration import ConfigurationVariable, \
+    configuration
+
+from typing import List, Optional, Tuple
+
+logger = logging.getLogger(__name__)
+
+
+def _get_aws_config() -> Tuple[str, dict]:
+    s3_region = configuration.get_value('AWS_DEFAULT_REGION')
+    s3_config = {
+        "aws_access_key_id": configuration.get_value('AWS_ACCESS_KEY_ID'),
+        "aws_secret_access_key": configuration.get_value(
+            'AWS_SECRET_ACCESS_KEY'),
+    }
+    return s3_region, s3_config
```

Comment:
Can't decide if this really needs to be in a separate function.

Reply:
I followed the style of what we do elsewhere to get the config (i.e. generate docs)